proofs/libsectorbuilder/cgo_bindings.go

```diff
@@ -49,6 +49,7 @@ type PieceMetadata struct {
 	Key            string
 	Size           uint64
 	InclusionProof []byte
+	CommP          [32]byte
```

Comment:
@ZenGround0 (cc @sidke) If you do introduce a constant, please leave the constant in `cgo_bindings.go`. Do not introduce a dependency on go-filecoin types and packages, as `cgo_bindings.go` and `cgo_transforms.go` are being extracted into a separate Git repository so that why's team can use them.